### DISCUSSION TITLE
Span.text?

### DISCUSSION THREAD

**Discussion by srikar-kodati** [explosion/spaCy#8015] [Help: Coding & Implementations] [answered]
Hey all! Im working on a project and building a parser. I have converted doc.sents into a list. The list contains spacy.tokens.span.Span. Any way i can convert these entries in the list to text (string)??

**Comment by polm** [comment, marked as answer]
Spans have [a text attribute](https://spacy.io/api/span#:~:text=a%20string%20representation%20of%20the%20span%20text.) you can use just like Docs. You can also call `str(span)` for the same output.
